relbench/tasks/event.py

```diff
@@ -81,8 +81,16 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
         event_attendees = db.table_dict["event_attendees"].df
         event_interest = db.table_dict["event_interest"].df
         timestamp_df = pd.DataFrame({"timestamp": timestamps})
+        eval_timestamp_len = len(timestamp_df)
```

Comment:
Update the doc-string to reflect the change

Reply:
Done